app/models/statbot.rb

```diff
@@ -0,0 +1,91 @@
+module StatBot 
+
+  def calculate_statistics
+    calculate_avg_ages
+    calculate_robots_hired_by_year
+    @max_hires = hires_by_year.max_by(&:last).last + 1 unless hires_by_year.empty?
+    @by_department = robots_by_department
+    @by_city = robots_by_city
+    @by_state = robots_by_state
+  end
+
+  def calculate_avg_ages
```

Comment:
this method is doing a great deal of logic. how can you extract some of this out to make this method more single responsibility? perhaps have individual methods that perform each of these calculations and make a call to them here